templates/web/components/schedule.tmpl

```diff
@@ -3,141 +3,228 @@
           </div>
         </div>
 
         <div class="bmvii-schedule__item">
           <div class="bmvii-schedule__item_info">
-            <span class="bmvii-schedule__item_time">9:30 PM</span>
-            <span class="bmvii-schedule__item_event">Dinner</span>
+            <span class="bmviii-schedule__item_time">11:00 pm</span>
+            <span class="bmviii-schedule__item_event">BM Family Feud</span>
           </div>
         </div>
 
         <div class="bmvii-schedule__item">
           <div class="bmvii-schedule__item_info">
-            <span class="bmvii-schedule__item_time">10:00 PM</span>
-            <span class="bmvii-schedule__item_event">Hacking Begins</span>
+            <span class="bmviii-schedule__item_time">12:00 am</span>
+            <span class="bmviii-schedule__item_event">Behind the Scenes Vlog</span>
           </div>
         </div>
 
         <div class="bmvii-schedule__item">
           <div class="bmvii-schedule__item_info">
-            <span class="bmvii-schedule__item_time">11:00 PM</span>
-            <span class="bmvii-schedule__item_event">Hackathon 101 w/ Zach Johnson</span>
+            <span class="bmviii-schedule__item_time">12:30 am</span>
+            <span class="bmviii-schedule__item_event">Music/Social Media Content</span>
           </div>
         </div>
 
+
       </div>
     </div>
   </div>
 
   <div class="column is-4">
     <div class="box bmvii-schedule__box">
-      <img alt="Saturday, January 25" class="bmvii-schedule__date" src={{ static_path "bmvii_schedule_saturday.svg" }}>
+      <img alt="Saturday, January 23" class="bmvii-schedule__date" src={{ static_path "bmviii_schedule_saturday.png" }}>
 
       <div class="bmvii-schedule__day">
+
         <div class="bmvii-schedule__item">
           <div class="bmvii-schedule__item_info">
-            <span class="bmvii-schedule__item_time">8:00 AM</span>
-            <span class="bmvii-schedule__item_event">Breakfast</span>
+            <span class="bmviii-schedule__item_time">9:30 am</span>
+            <span class="bmviii-schedule__item_event">Announcements</span>
           </div>
         </div>
 
         <div class="bmvii-schedule__item">
           <div class="bmvii-schedule__item_info">
-            <span class="bmvii-schedule__item_time">10:00 AM</span>
-            <span class="bmvii-schedule__item_event">John Deere Tech Talk</span>
+            <span class="bmviii-schedule__item_time">10:00 am</span>
+            <span class="bmviii-schedule__item_event">Tech Talk: Ecolab</span>
           </div>
         </div>
 
         <div class="bmvii-schedule__item">
           <div class="bmvii-schedule__item_info">
-            <span class="bmvii-schedule__item_time">11:00 AM</span>
-            <span class="bmvii-schedule__item_event">Qualcomm Tech Talk</span>
+            <span class="bmviii-schedule__item_time">11:00 am</span>
+            <span class="bmviii-schedule__item_event">Sponsor Interview: Ecolab</span>
```

Comment:
Change space between : and Ecolab to <br>, else the line is a bit clunky.

Reply:
Can you send a screenshot? This is what it looks on my screen. 
![Screen Shot 2021-01-18 at 11 16 14 AM](https://user-images.githubusercontent.com/31838229/104939588-9b3ab780-597e-11eb-86aa-3a6d5ba421d5.png)